folium/folium.py

```diff
@@ -466,14 +469,17 @@ def choropleth(self, geo_data, data=None, columns=None, key_on=None,
             start with 'feature' and be in JavaScript objection notation.
             Ex: 'feature.id' or 'feature.properties.statename'.
         threshold_scale: list, default None
```

Comment:
Not your change, but I would like that all arguments that have a default value of `None` because they are optional to have a docstring like `name: type, optional` instead of `name: type, default None`. In this case the default is actually a list with 7 values.

Reply:
I didn't get what you are saying here, could you clarify a bit? Thanks

Reply:
In many cases the default value is None, but from the users perspective that's not really helpful since None disables the argument. In that case it's better to say the argument is optional.

But here that's not really the case since the default value should actually be a list of 7 values so I guess my point doesn't really work here.

This doesn't matter here so I'll set the conversation as resolved :)